SPECS/kata-containers-cc/kata-containers-cc.spec

```diff
@@ -17,6 +17,8 @@ Source0:      https://github.com/microsoft/kata-containers/archive/refs/tags/cc-
 Source1:      https://github.com/microsoft/kata-containers/archive/refs/tags/%{name}-%{version}.tar.gz
 Source2:      %{name}-%{version}-cargo.tar.gz
 Source3:      mariner-coco-build-uvm.sh
+Patch0:       0001-tardev-snapshotter-enable-feature-impl_trait_in_asso.patch
+Patch1:       drop-mut-for-variables-tahat-are-not-mutated.patch
```

Comment:
A more general question: Was there a build issue on new rust version with unmated mut variables?

Reply:
See the error logs I posted below.